.gitignore

```diff
@@ -2,3 +2,4 @@
 /vendor
 /vendor-bin
 /composer.lock
+/.idea
```

Comment:
Why not use a global ignore file?

See https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer.

Reply:
Could also work though i expect not many ppl to use it dont you think?